plugins/generator-1.20.1/forge-1.20.1/mappings/noteblockinstruments.yaml

```diff
@@ -0,0 +1,51 @@
+_default:

```

Comment:
I have checked 1.20.1 again, and it seems both versions have all the elements as in 1.21.1. Not sure how we came to conclusion that 1.20.1 does not have all of them, but the reality is the list was missing 1.20.1 entries from the start, meaning we don't even need those mappings yet in reality

Reply:
Keep the system in place, but remove noteblock mappings from this PR and inline back list of noteblocks, but make sure to include all of them

Reply:
✔️ Fixed